modules/create-a-containerruntimeconfig-crd.adoc

```diff
@@ -15,11 +15,10 @@ To revert the changes implemented by using a `ContainerRuntimeConfig` CR, you mu
 
 You can modify the following settings by using a `ContainerRuntimeConfig` CR:
 
-* **PIDs limit**: The `pidsLimit` parameter sets the CRI-O `pids_limit` parameter, which is maximum number of processes allowed in a container. The default is 1024 (`pids_limit = 1024`).
+* **PIDs limit**: The `--pod-pids-limit` Kubelet flag sets the CRI-O `pids_limit` parameter, which is maximum number of processes allowed in a container. The default is 0 (`pids_limit = 0`).
 * **Log level**: The `logLevel` parameter sets the CRI-O `log_level` parameter, which is the level of verbosity for log messages. The default is `info` (`log_level = info`). Other options include `fatal`, `panic`, `error`, `warn`, `debug`, and `trace`.
 * **Overlay size**: The `overlaySize` parameter sets the CRI-O Overlay storage driver `size` parameter, which is the maximum size of a container image.
-* **Maximum log size**: The `logSizeMax` parameter sets the CRI-O `log_size_max` parameter, which is the maximum size allowed for the container log file. The default is unlimited (`log_size_max = -1`). If set to a positive number, it must be at least 8192 to not be smaller than  the ConMon read buffer. ConMon is a program that
-monitors communications between a container manager, such as Podman or CRI-O, and the OCI runtime, such as runC or crun, for a single container.
+* **Maximum log size**: The `--container-log-max-size` Kubelet flag sets the CRI-O `log_size_max` parameter, which is the maximum size allowed for the container log file. The default is unlimited (`log_size_max = -1`). If set to a positive number, it must be at least 8192 to not be smaller than  the ConMon read buffer. ConMon is a program that monitors communications between a container manager, such as Podman or CRI-O, and the OCI runtime, such as runC or crun, for a single container.
```

Comment:
Same here: another suggested reword:

```
Setting maximum log size in the `ContainerRuntimeConfig` is in the process of being deprecated. If a maximum log size is required, the recommendation is to use the `KubeletConfig` field `ContainerLogMaxSize` instead.
```

Reply:
@haircommander Should I include this sentence as well? 
> If set to a positive number, it must be at least 8192 to not be smaller than  the ConMon read buffer. ConMon is a program that monitors communications between a container manager, such as Podman or CRI-O, and the OCI runtime, such as runC or crun, for a single container.

Reply:
I don't believe the kubelet is limited in the same way

Reply:
Removed it then, thanks